app/controllers/posts_controller.rb

```diff
@@ -4,7 +4,7 @@ class PostsController < ApplicationController
   before_action :require_login, except: [:index, :show]
 
   def index
-    @posts = @user.posts.all
+    @posts = @user.posts.order("created_at").page(params[:page]).per(20)
```

Comment:
Done. The file is config/initializers/kaminari.rb. Does the naming matter? I'll change the tests to reflect the changes.